src/main/java/core/basesyntax/model/FruitsForSale.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.model;
+
+import java.util.List;
+
+public class FruitsForSale {
+    private static final List<String> fruitsForSale
+            = List.of("banana", "apple");
+
+    public boolean isNotForSale(String fruit) {
+        for (String fruitFoSale: fruitsForSale) {
```

Comment:
There is a typo in the variable name `fruitFoSale`. It should be `fruitForSale` for better readability and consistency.